In [1]:
x = 25
y = 20

result = x*y
print (result)

500


I created variables x and y, assigned 25 to x and 20 to the value of y, created a function to multiply them and printed the result


In [6]:
if result > 200:
    print ("You beat the high score!")
else:
    print ("Try again, you failed to beat the high score")

You beat the high score!


This checks if the score is above 200, if its higher than 200 it beats the high score and prints the message, if its any number below that it sayts to try again

In [8]:
result = 500
for i in range(500, 510):
    result = result + i
    print("i =", i, "updated results=", result)
print("Final result:", result)

i = 500 updated results= 1000
i = 501 updated results= 1501
i = 502 updated results= 2003
i = 503 updated results= 2506
i = 504 updated results= 3010
i = 505 updated results= 3515
i = 506 updated results= 4021
i = 507 updated results= 4528
i = 508 updated results= 5036
i = 509 updated results= 5545
Final result: 5545


The loop starts with the reuslt of 500 and then it iterates through numbers 500 to 509 and on each pass adds the former value of the result (500, 501, 502) to the total. for loops, it would be better if i  created a cleaner example of numbers 1-10 so it just gradually builds but it gets incrementally larger because of the addition of each score